README.md

```diff
@@ -158,16 +160,13 @@ Available as official Arch Linux package. Install it via:
 sudo pacman -S kubectx
 ```
 
-#### Homebrew
-
-:confetti_ball: If you use [Homebrew](https://brew.sh/) you can install like this:
-
-    brew install kubectx
+#### Chocolatey
```

Comment:
It seems you are

1. removing linux brew instructions
2. missing a `### Windows` title.

once we fix those, we can merge this.

Reply:
thanks for catching that, I believe I have corrected both of those. Let me know if anything else is out of place and I'll try to correct it as well.